.github/workflows/sqlalchemy-dialect.yml

```diff
@@ -0,0 +1,178 @@
+name: SQLAlchemy dialect test
+on: 
+  push:
+    branches:
+      - sqlalchemy-dev
+    paths:
+      - Makefile
+      - src/databricks/sqlalchemy
+      - tests/sqlalchemy
+      - .github/workflows/sqlalchemy-dialect.yml
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      
+      #----------------------------------------------
+      #       check-out repo and set-up python
+      #----------------------------------------------
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Set up python
+        id: setup-python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
does this work on other versions of python3 as well? any limitation on the supported python version?

Reply:
It should. Note that I cribbed from code-quality-checks.yml for this action to test out some ideas. Now that I think about it, this action won't be operative since I've since removed the Pytest test. I will remove this action completely.

(The current plan is to *not* turn on Github Action in the OSS repo; contributors are encouraged to do so and to use these though.)

Reply:
Rectified